tests/main/apparmor-prompting-client-integration-tests/task.yaml

```diff
@@ -0,0 +1,76 @@
+summary: Run the prompting-client integration tests against snapd
+
+details: |
+    This test checks that the integration tests for the prompting-client pass
+    when run against snapd.
+
+systems:
+  - ubuntu-2*
+
+prepare: |
+    # Prerequisite for having a prompt handler service
+    snap set system experimental.user-daemons=true
+    "$TESTSTOOLS"/snaps-state install-local test-snapd-prompt-handler
+    snap connect test-snapd-prompt-handler:snap-interfaces-requests-control
+
+    snap install lxd
+    lxd init --auto
+    snap install snapcraft --classic
+
+    git clone https://github.com/canonical/prompting-client /home/ubuntu/prompting-client
```

Comment:
about this, is it possible to upload the snap to the snapt and avoid using snapcraft in the test?

Reply:
Ideally we'd build the snap and integration tests binary whenever a new release of the prompting-client occurs, but this is not being done at the moment. I'll bring it up with the Desktop team again, as these do take a very long time in spread.